systemdspawner/systemd.py

```diff
@@ -143,6 +144,20 @@ async def start_transient_service(
         )
         run_cmd.append(f"--property=EnvironmentFile={environment_file}")
 
+    # make sure cmd[0] is absolute, which is a requirement of systemd-run
+    # if it's relative, resolve it with Spawner's $PATH,
+    # as any other Spawner would behave
+    if not os.path.isabs(cmd[0]):
+        path = os.getenv("PATH", os.defpath)
+        path = (environment_variables or {}).get("PATH")
+        if environment_variables and "PATH" in environment_variables:
+            path = environment_variables["PATH"]
```

Comment:
I think `path = (environment_variables or {}).get("PATH")` on the line above accomplishes this anyhow, right?

```suggestion
```

Reply:
Yes, but it's actually the _previous_ line that's wrong, not this one.